plugins/thunderbolt/fu-self-test.c

```diff
@@ -338,6 +348,55 @@ mock_tree_node_have_fu_device (const MockTree *node, gpointer data)
 	return node->fu_device != NULL;
 }
 
+static void
+write_controller_fw (const gchar *nvm)
+{
+	g_autoptr(GFile) nvm_device = NULL;
+	g_autoptr(GFile) nvmem = NULL;
+	g_autofree gchar *fw_path = NULL;
+	g_autoptr(GFile) fw_file = NULL;
+	g_autoptr(GInputStream) is = NULL;
+	g_autoptr(GOutputStream) os = NULL;
+	g_autoptr(GError) error = NULL;
+	gssize n;
+	const gssize header_size = 0x4000;
+
+	fw_path = fu_test_get_filename (TESTDATADIR, "thunderbolt/firmware.bin");
+	g_assert_nonnull (fw_path);
```

Comment:
Rather than fail the test suite on this, could you instead skip the test if the file isn't found (per other discussion in this thread)?

Reply:
Better, I'm changing it to use the minimal "FW" I have just created. This file shouldn't have any license issues. Makes sense?